### PR TITLE
feat: Adding docs on new Pipelines configs

### PR DESCRIPTION
**Pull request by yhakbar**
Adds documentation on recently added Pipelines configurations.

### CODE REVIEW

File: _docs-sources/pipelines/overview/configurations-as-code.md [hunk 1]
```diff
@@ -254,6 +254,12 @@ In this example, the `deploy_branch_name` attribute is set to `main`, which mean
 *Supported Attributes:*
 
 - `deploy_branch_name` (Optional): The branch that Pipelines will deploy infrastructure changes from. If not set, Pipelines will deploy infrastructure changes from the `main` branch.
+- `consolidate_added_or_changed` (Optional): Whether or not Pipelines will consolidate added or changed resources when running Terragrunt commands. If not set, Pipelines will consolidate added or changed resources.
```
Comment (ZachGoldberg) [inline _docs-sources/pipelines/overview/configurations-as-code.md:257]: I'm not sure users know what "consolidated" means, probably worth clarifying multiple jobs vs. `run-all`

Comment (ZachGoldberg) [reply _docs-sources/pipelines/overview/configurations-as-code.md:257]: Thinking about it... this is a big topic, maybe we should link to a dedicated docs section about consolidation?

Comment (yhakbar) [reply _docs-sources/pipelines/overview/configurations-as-code.md:257]: Is this good enough for now?
https://github.com/gruntwork-io/docs/pull/1859/commits/0686f422ad272d5bb8fd77557627fbee44e91dd0

We can revisit later?

**Review by ZachGoldberg — APPROVED** [review]

**Review by bendavies — COMMENTED** [review]
this is great. I had this exact problem where pipelines could not plan my changes due to dependent modules.

**Pull request merged**